relay-server/src/metrics_extraction/transactions.rs

```diff
@@ -198,11 +211,11 @@ mod tests {
             r#"
         {
             "extractMetrics": [
-                "measurements.foo",
-                "measurements.lcp",
-                "breakdown.breakdown1.bar",
-                "breakdown.breakdown2.baz",
-                "breakdown.breakdown2.zap"
+                "sentry.transactions.measurements.foo",
+                "sentry.transactions.measurements.lcp",
+                "sentry.transactions.breakdowns.breakdown1.bar",
+                "sentry.transactions.breakdowns.breakdown2.baz",
+                "sentry.transactions.breakdowns.breakdown2.zap"
```

Comment:
I briefly considered omitting the prefixes in the project config, since we're already in a config object called `transactionMetrics`. But I figured that listing the full metric name would simplify searching for it, as with snuba referrers.